catalog/cataloger_get_commit_test.go

```diff
@@ -97,6 +93,9 @@ func TestCataloger_GetCommit(t *testing.T) {
 				t.Errorf("GetCommit() error = %s, wantErr %t", err, tt.wantErr)
 				return
 			}
+			if got != nil {
+				got.CreationDate = got.CreationDate.Truncate(time.Minute)
```

Comment:
same as before - don't want to play sync my clock with the database - prefer to skip the creation time or just verify once it is ~ the same time and not truncate and have the test fail once when we cross the minute.

Reply:
Yup, as above.